cpp/src/parquet/statistics.cc

```diff
@@ -495,9 +495,12 @@ class TypedStatisticsImpl : public TypedStatistics<DType> {
                       bool has_null_count, bool has_distinct_count, MemoryPool* pool)
       : TypedStatisticsImpl(descr, pool) {
     TypedStatisticsImpl::IncrementNumValues(num_values);
+    // Currently, `has_null_count` argument is not used.
+    // Internal has_null_count_ would always be true.
```

Comment:
IMO, if `has_null_count` is not used we should remove it to reduce confusion and future misuse. Otherwise, we should respect it from the input argument.

Reply:
I agree with you, but I need other's opinion

Reply:
IIUC, this class is only used in the writer and we always write the null count in statistics? If so, then I agree we can remove this.

Reply:
After a weird patch, this class is used in reader, but ndv is not used.

Reply:
what is ndv?

Reply:
Sorry for make misunderstanding. NDV means num-different values. Load statistics is introduced in `ColumnChunkMetaData`, and it's weird that:

1. Writer will never set ndv.
2. Reader should read ndv.

The behavior here is trickey